docs/shared-content-source/docs/modules/develop/pages/blueprints.adoc

```diff
@@ -18,57 +20,204 @@ include::{cloudflow-examples-version}@docsnippets:ROOT:example$akkastreams-scala
+
+TIP: Using the configuration system, different streamlet instances of the same implementation can be given a different configuration so that they can work in customized ways.
+
+=== Understanding the `short-name`
+
+There are a few characteristics related to the streamlet name given with the `short-name`, as we list here:
+
+- The short name gives a name to a streamlet instance.
+- The implementation of that instance is indicated by the fully qualified name (FQN) of the streamlet implementation class. 
+- Each short name must be unique within the application.
+
+The `short-name` becomes part of the application in the following ways:
+
+- We use this name to refer to the streamlets in the _topics_ section.
+- It becomes the name of the component in the application runtime (either on a cluster or on the local sandbox)
+
+=== The `streamlets` Section: An Example 
+
+Lets zoom in on the streamlets section in the example below:
 
 [source,hocon]
 ----
 include::{cloudflow-examples-version}@docsnippets:ROOT:example$akkastreams-scala/src/main/blueprint/blueprint.conf[tag=streamlets]
 ----
 
-The above example shows four streamlet types defined in the `cloudflow.examples.sensordata` package.
-Streamlet types (classes or objects) are specified by their fully qualified names.
+The above example shows four streamlet instances defined in the `com.example` package:
 
-Every assignment in the `streamlets` section assigns a streamlet reference to an instance of a type of streamlet. Once deployed, Cloudflow will run at least one streamlet–depending on the requested number of replicas–for each streamlet reference that has been defined in the blueprint.
+- `sensor-data`
+- `metrics`
+- `validation`
+- `reporter`
 
-The Streamlets can be defined in Java or Scala,
-as Java or Scala classes with default constructors or as Scala objects. These classes must be available on the classpath, which can be defined directly in the Cloudflow application sbt project or in any dependency that the project has specified, as you would expect of any sbt project.
+Streamlet classes are specified by their fully qualified names:
 
-[NOTE]
-====
-You can define more than one streamlet reference to the same Scala object.
-In that case, as many streamlets as assigned streamlet references will be run once everything is deployed (assuming a scaling factor of one). You should view a Streamlet, defined as a Scala object as a template that can be used many times to instantiate streamlets, not as a singleton object that will only run once as a streamlet.
-====
+[source, json]
+----
+sensor-data = com.example.SensorDataIngress
+----
 
-The streamlet references assigned in the `streamlets` section can be used in the `connections` section to connect streamlets together.
-Lets zoom in on the streamlets section in the example below:
+It is possible to declare additional instances of any streamlet class. 
+This would be a legal declaration:
 
-[source,hocon]
+[source, json]
```

Comment:
Why json? The file content is in hocon.

Reply:
I was trying to see if the rendering (coloring) was better. Changed back.